Pipelines/pr.yaml

```diff
@@ -29,3 +34,16 @@ jobs:
     parameters:
       BuildStandalone: true
       BuildUWP: true
+
+- job: DocsBinariesGeneration
+  pool: Unity_2021.3.3f1_Pool
+  steps:
+  - checkout: self
+
+  - checkout: PipelineTools
```

Comment:
What's used in this job from the `PipelineTools` repo?

Reply:
Almost everything related to docs gen minus the yaml files (as you can see I am only checking in yaml files in this PR). We are exporting a unity package from that repo during CI time (using a script in that repo), and then import the exported package into the MRTK3 Unity project where we run the command to generate the binaries (also using a script in that repo).

Reply:
I might be getting this backwards, but I thought the docs gen stuff was all in the DocToolUnityProject repo

Reply:
Oh yes you are right, the docs gen stuff was in the lib.doctools repo. However, we still need the pipeline tools repo to set up Unity license correctly (please refer to line 47 below). Without that I ran into Unity licensing issues.